librz/core/cmd/cmd_analysis.c

```diff
@@ -1791,7 +1792,7 @@ RZ_IPI RzCmdStatus rz_analysis_syscall_print_handler(RzCore *core, int argc, con
 		switch (state->mode) {
 		case RZ_OUTPUT_MODE_STANDARD:
 			rz_cons_printf("%s = 0x%02x.%s\n",
-				si->name, si->swi, syscallNumber(si->num));
+				si->name, si->swi, rz_strf(tmpbuf, syscallNumberFmt(si->num), si->num));
```

Comment:
merge `syscallNumberFmt` in here. there is no reason to do double formatting.

Reply:
There is no easy way to do that because C doesn't allow nested formatting or runtime string resolution.